File: test/fixtures/scripts/hello.py
"""
A simple hello world script for testing the Raxol terminal emulator.
"""


def hello(name="World"):
    return f"Hello, {name}!"


def run(args=None):
    if args is None:
        args = []
    name = args[0] if args else "World"
    return hello(name)

File: test/fixtures/scripts/hello.ex
defmodule Raxol.Terminal.Script.Hello do
  @moduledoc '''
  A simple hello world script for testing the Raxol terminal emulator.
  '''

  def hello(name \\ "World") do
    "Hello, #{name}!"
  end

  def run(args \\ []) do
    name = List.first(args) || "World"
    hello(name)
  end
end
